test/helper.js

```diff
@@ -36,9 +37,18 @@ const helper = module.exports = {
           fs.mkdirSync(path);
         }
       });
+
+      migrationsPath = path;
+      if (options.usesSymbolicRef) {
+        const dirName = randomName();
+        migrationsPath = join(__dirname, '/tmp', dirName, '/');
+        path = join(path, dirName);
+        fs.mkdirSync(migrationsPath);
+        fs.symlinkSync(migrationsPath, path);
+      }
     }
     fs.writeFileSync(
-      join(path, name + '.js'),
+      join(migrationsPath, name + '.js'),
```

Comment:
observe the use of migrationsPath here, which could be pointing to any external directory or plugins